OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -5,30 +5,29 @@
   
   Compile options:
     ESP32
-
   External dependencies. Install using the Arduino library manager:
     "Adafruit_MCP23X17" (requires recent "Adafruit_BusIO" library)
     "PubSubClient" by Nick O'Leary
     "OXRS-IO-MQTT-ESP32-LIB" by OXRS Core Team
     "OXRS-IO-LCD-ESP32-LIB" by OXRS Core Team
     "OXRS-SHA-IOHandler-ESP32-LIB" by SuperHouse Automation Pty
-
   Compatible with the Light Switch Controller hardware found here:
     www.superhouse.tv/lightswitch
-
   GitHub repository:
     https://github.com/SuperHouse/OXRS-SHA-StateMonitor-ESP32-FW
-

```

Comment:
Any reason you removed this whitespace? I think it makes the header a bit easier to read with some lines in between sections.

Reply:
not by intention